.github/workflows/docker-build-test-upload.yml

```diff
@@ -41,7 +41,7 @@ jobs:
     steps:
       # Image with CUDA needs extra disk space
       - name: Free disk space 🧹
-        if: contains(inputs.variant, 'cuda') && inputs.platform == 'x86_64'
+        if: contains(inputs.variant, 'cuda') && runner.arch == 'X64'
```

Comment:
Let's use `inputs.platform` when we want to name things
`runner.arch` is better, because it's standard when using GitHub runners

I will merge this change separately, so it's not part of this PR